src/_rules/color.js

```diff
@@ -1,5 +1,15 @@
-import { handler as h } from '#utils';
+
+const handleOpacity = ([, d = ''], { theme }) => {
```

Comment:
Have you checked if this can be handled down in the tuple as this shortened version?

```
([, d]) => ({ opacity: theme.opacity[d] })
```

Reply:
(Had a free moment so I checked this and all tests pass!)